backend/domain/MafiaGame.js

```diff
@@ -1,19 +1,46 @@
+const Room = require("./Room");
+
 class MafiaGame {
     constructor() {
-        // Acts  as a mapping for gameID keys to their gameroom.
-        const gameRoomsDict = {};
+        // Acts as a mapping for roomID keys to their game room.
+        this.gameRoomsDict = {};
     }
 
+    /**
+     * Create a new game by creating a new room
+     * @returns roomID : String - The newly generated room.
+     */
     newGame() {
-        let newRoom = new Room();
-        let gameID = newRoom.getRoomName();
-        gameRoomsDict[gameID] = newRoom;
+        // Create a new room and add it to existing rooms
+        var room = this.createNewRoom();
```

Comment:
Var is fine here, but was there a scope issue with "Let"?
Don't want to change it back or anything just want to know for future reference.

Reply:
I thought scope issues arise with var and that’s why it’s bad practice ?

Reply:
`let` should still work in this case.